Documentation/codereview-policy.md

```diff
@@ -0,0 +1,45 @@
+### Code Review Policy
+
+#### How to review pull requests  
+
+This is a guide for core developers and contributors who want to help with code reviewing PRs on the fullnode.  
+We welcome anyone to help with code reviews, from typos to suggestions in design.   
+
+#### Why are code reviews important?  
+It's essential that PRs are properly code-reviewed.   
```

Comment:
This line isn't an answer to the question above and could be removed merged into the title... eg
`#### Why it is essential that PRs are properly code-reviewed?`